frontend/lib/src/util/UriUtil.test.ts

```diff
@@ -190,6 +190,30 @@ describe("isValidOrigin", () => {
     ).toBeTruthy()
   })
 
+  it("returns false if testUrl's hostname is localhost but protocol doesn't match", () => {
+    expect(isValidOrigin("http://localhost", "https://localhost")).toBeFalsy()
+
+    expect(
+      isValidOrigin("https://localhost:8000", "http://localhost:8000")
+    ).toBeFalsy()
```

Comment:
@sfc-gh-bnisco Makes sense. I mostly duplicated what other assertions in this file are using. Since we're supposed to always get a boolean here, should I update all assertions to `.toBe(true)` and `.toBe(false)`, not just the ones I added/updated?

Reply:
Thanks! I'd be in favor of updating all of them for consistency.

Reply:
Updated.